docs/undergrad.yaml

```diff
@@ -67,7 +67,7 @@ Linear algebra:
     Jordan normal form: 'https://en.wikipedia.org/wiki/Jordan_normal_form'
   Linear representations:
     irreducible representation: 'https://en.wikipedia.org/wiki/Irreducible_representation'
-    Schur's lemma: 'https://en.wikipedia.org/wiki/Schur%27s_lemma'
+    Schur's lemma: `linear_map.bijective_or_eq_zero`, 'category_theory.finrank_endomorphism_simple_eq_one'
```

Comment:
We still don't have the "canonical" version for irreducible representations, right?  I think we had considered that this entry should wait for that.

Reply:
Will trust you on that one!